<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>9</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color=rgb(211, 215, 207)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Clock ticks per second</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>40</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>more ticks
per second
simulates
faster CPU</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>20</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>This demo  
by Jim Stallings
shows how to run 
a clock in the background, 
and dynamically speed it up 
or slow it down.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By James Stallings Oct 5, 2019</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>226</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>120</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="ClockTicksText">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Get clock ticks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ticks  elepsed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>frame</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
